jsapp/js/editorMixins/assetNavigator.es6

```diff
@@ -95,13 +95,14 @@ class AssetNavigatorListView extends React.Component {
       window.setTimeout(()=>{
         this.activateSortable();
       }, 1);
-
       return (
         <bem.LibList m={['done', isSearch ? 'search' : 'default']} ref='liblist'>
           {list.map((item)=> {
             var modifiers = [item.asset_type];
             var summ = item.summary;
-            if (summ.row_count == undefined) {
+            // Library questions can only be of `question` or `block` types
+            // Reject `survey` types to not include current survey on save
+            if (summ.row_count == undefined || item.asset_type == 'survey') {
```

Comment:
Please use `ASSET_TYPES.survey.id` from `constants.es6`

Reply:
And please add a comment stating that this is a `HACK FIX`, as ideal solution would be to not search for `survey`s in the first place - but I understand that it would require too much work influencing too many places in the code

Reply:
Oh and please use triple equals to check for `asset_type` :)